suite-common/wallet-utils/src/filterReceiveAccounts.ts

```diff
@@ -15,6 +15,16 @@ export const isDebugOnlyAccountType = (
     return !!accountTypeInfo?.isDebugOnlyAccountType;
 };
 
+const accountTypeOrder: Record<AccountType, number> = {
+    normal: 0,
+    ledger: 1,
+    legacy: 2,
+    imported: 3,
+    segwit: 4,
+    coinjoin: 5,
+    taproot: 6,
+};
```

Comment:
I guess Bitcoiners will not be happy to have `legacy` accounts before `segwit` and `taproot`

I would expect something like:

1. normal (BTC, ETH, SOL,...)
2. taproot (BTC)
3. segwit (BTC)
4. coinjoin (BTC)
5. ledger (ETH, SOL)
6. legacy (BTC, ETH)
7. imported (??? mobile)

Please someone confirm @hynek-jina @marekrjpolak

Reply:
I would stick to that and reuse it
![Screenshot 2025-01-14 at 15 41 39](https://github.com/user-attachments/assets/f4e45b01-7b9a-46c8-9719-d640245a2ba4)

Reply:
resolved